incubator/hnc/internal/reconcilers/object_test.go

```diff
@@ -45,30 +45,46 @@ var _ = Describe("Secret", func() {
 		cleanupObjects(ctx)
 	})
 
-	// Exceptions have not been implemented, so we are ignoring the following three tests for now.
+	// Exceptions have not been implemented, so we are ignoring the following four tests for now.
 	// They should be enabled once exception is done.
 	PIt("should propagate object only to selected namespace using treeSelect", func() {
 		setParent(ctx, barName, fooName)
 		setParent(ctx, bazName, fooName)
 
 		// Create a secret that does NOT propogate to bazName
 		a := map[string]string{"propagate.hnc.x-k8s.io/treeSelect": "!" + bazName}
-		makeObjectWithAnnotation(ctx, "Secret", fooName, "fooSecret", a)
+		makeObjectWithAnnotation(ctx, "Role", fooName, "testrole", a)
 
-		Eventually(hasObject(ctx, "Secret", barName, "fooSecret")).Should(BeTrue())
-		Consistently(hasObject(ctx, "Secret", bazName, "fooSecret")).Should(BeFalse())
+		Eventually(hasObject(ctx, "Role", barName, "testrole")).Should(BeTrue())
+		Consistently(hasObject(ctx, "Role", bazName, "testrole")).Should(BeFalse())
 	})
 
 	PIt("should propagate object only to selected namespace using select", func() {
 		setParent(ctx, barName, fooName)
 		setParent(ctx, bazName, fooName)
 
 		// Create a secret that does NOT propogate to bazName
```

Comment:
Seems the "secret"s are not replaced by "role"s in the comments.

Reply:
It should be updated now